sensor_msgs/Image rawRGB
sensor_msgs/PointCloud2 rawDepth
sensor_msgs/PointCloud2 segmented
sensor_msgs/PointCloud2[] clustered
vfh[] vfhs
string[] label
float32[] score
